### PR TITLE
Enable allowClear as a dynamic property for dropdowns

### PR DESCRIPTION
**Pull request by abhilashlr89**

### CODE REVIEW

File: addon/components/form-controls/power-select/component.js [hunk 1]
```diff
@@ -79,6 +81,11 @@ const PowerSelectComponent = Ember.Component.extend({
     }
   }),
 
+  shouldAllowClear: computed({
```
Comment (parsurv) [inline addon/components/form-controls/power-select/component.js]: shouldAllowClear: computed.bool('allowClear');

Will this be easier.

Comment (abhilashlr89) [inline addon/components/form-controls/power-select/component.js]: Allowclear should be true by default even if allowClear is undefined. 

If computed.bool() is used, for undefined state it becomes false, and there by making all dropdowns lose that clear action. 

Should we take the stance of keeping false by default @parsurv ? If so, can do it as a simple computed.

Comment (parsurv) [inline addon/components/form-controls/power-select/component.js]: Yes keep allowClear as false

Comment (abhilashlr89) [inline addon/components/form-controls/power-select/component.js]: Alright @parsurv. Will do that.

**Pull request merged**